certificate-policy.md

```diff
@@ -1760,9 +1760,8 @@ The CA's audit SHALL be performed by a Qualified Auditor. A Qualified Auditor me
 2. The ability to conduct an audit that addresses the criteria specified in an Eligible Audit Scheme (see Section 8.1);
 3. Employs individuals who have proficiency in examining Public Key Infrastructure technology, information security tools and techniques, information technology and security auditing, and the third-party attestation function;
 4. (For audits conducted in accordance with the WebTrust standard) licensed by WebTrust;
-5. Bound by law, government regulation, or professional code of ethics;
-6. Except in the case of an Internal Government Auditing Agency, maintains Professional Liability/Errors & Omissions insurance with policy limits of at least one million US dollars in coverage; and
-7. If the CA chooses to obtain a WebTrust audit, the CA SHALL use a WebTrust licensed Auditor to perform the audit. 
+5. Bound by law, government regulation, or professional code of ethics; and
+6. Except in the case of an Internal Government Auditing Agency, maintains Professional Liability/Errors & Omissions insurance with policy limits of at least one million US dollars in coverage.
```

Comment:
Is this saying we would not obtain a WebTrust audit? I think we should welcome a third party assessor.